client/nginx.go

```diff
@@ -30,7 +30,7 @@ type UpstreamServer struct {
 	ID          int    `json:"id,omitempty"`
 	Server      string `json:"server"`
 	MaxConns    int    `json:"max_conns"`
-	MaxFails    int    `json:"max_fails"`
+	MaxFails    int    `json:"max_fails,omitempty"`
```

Comment:
this won't work. `max_fails=0` is a valid value. if we have `omitempty` here, the user will never be able to set `max_fails` to `0`.

Reply:
My mistake, should be fixed now.